docs/.vitepress/theme/components/Feedback/Feedback.vue

```diff
@@ -14,16 +15,23 @@ enum FeedbackState {
 const route = useRoute();
 const currentState = ref(FeedbackState.START);
 const feedback = ref('');
+const feedbackTextarea = ref(null);
 const isSending = ref(false);
 const hasError = ref(false);
 
 function onButtonClick(value: FeedbackState.YES | FeedbackState.NO) {
+  // don't change state if opposite button is active.
+  const opposingState = value === FeedbackState.YES ? FeedbackState.NO : FeedbackState.YES;
+  if ([opposingState].includes(currentState.value)) {
+    return;
+  }
   currentState.value = value;
-  sendEvent({
-    eventName: 'wc_doc_feedback_helpful',
-    pagePath: route.path,
-    value,
-  });
+  setTimeout(() => document.getElementById('feedback')?.focus());
```

Comment:
This was the only way I could get the `textarea` focused after the visibility state change, though I'm not as familiar with Vue.

@ykmsd I think you made this component initially, any ideas?

Reply:
That seems to do it, thanks @ykmsd!

Though I'm getting some squiggly lines, but my Volar plugin isn't working correctly in general, so it could be that.
No errors in the console, and it does focus correctly 🤷 

![Screen Shot 2023-03-15 at 12 09 31 PM](https://user-images.githubusercontent.com/22125230/225403838-1fdc5b16-7ee5-479a-ae96-73d0edb27491.png)

https://user-images.githubusercontent.com/22125230/225403982-2a146943-bdf0-4f14-bea3-63388cf79ed6.mov